service/history/api/getworkflowexecutionhistory/api.go

```diff
@@ -180,6 +183,12 @@ func Invoke(
 		continuationToken.PersistenceToken = nil
 	}
 
+	// TODO: remove this after it start to support it
+	if clientName, _ := headers.GetClientNameAndVersion(ctx); clientName == headers.ClientNameCLI || clientName == headers.ClientNameUI {
+		// Pretend that there is transient or speculative WFT if client is CLI or UI because they don't support it.
```

Comment:
Do you miss a NO in the comment?

Reply:
![](https://user-images.githubusercontent.com/2232524/75076254-832f8f00-54b4-11ea-81b9-1a6648a486a5.png)